### PR TITLE
Update plugin base and logging

### PR DESCRIPTION
**Pull request by dmdashenkov**
In this PR we update the version of `plugin-base` and update config.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol, @alexander-yevsyukov, PTAL.

File: buildSrc/build.gradle.kts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2023, TeamDev. All rights reserved.
+ * Copyright 2022, TeamDev. All rights reserved.
```
Comment (armiol) [inline buildSrc/build.gradle.kts:2]: Something is mixed up here. It's 2023.

File: buildSrc/src/main/kotlin/io/spine/internal/dependency/ProtoData.kt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022, TeamDev. All rights reserved.
+ * Copyright 2023, TeamDev. All rights reserved.
```
Comment (armiol) [inline buildSrc/src/main/kotlin/io/spine/internal/dependency/ProtoData.kt:2]: And here.

File: buildSrc/src/main/kotlin/io/spine/internal/gradle/publish/JarDsl.kt [hunk 3]
```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright 2022, TeamDev. All rights reserved.
```
Comment (armiol) [inline buildSrc/src/main/kotlin/io/spine/internal/gradle/publish/JarDsl.kt:2]: And here.

File: buildSrc/src/main/kotlin/io/spine/internal/gradle/publish/PublishingExts.kt [hunk 4]
```diff
@@ -0,0 +1,264 @@
+/*
+ * Copyright 2022, TeamDev. All rights reserved.
```
Comment (armiol) [inline buildSrc/src/main/kotlin/io/spine/internal/gradle/publish/PublishingExts.kt:2]: Here as well.

**Review by armiol — CHANGES_REQUESTED** [review]
@dmdashenkov LGTM in general, but could you please update the copyright gear in `config` first? Right now it's a bit messy.

**Pull request merged**